api/v1beta1/neutronapi_types.go

```diff
@@ -135,6 +135,10 @@ type NeutronAPISpecCore struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// TLS - Parameters related to the TLS
 	TLS NeutronApiTLS `json:"tls,omitempty"`
+
+	// +kubebuilder:validation:Required
+	// NeutronAPI Route Timeout (will be set to environmental default if empty)
+	NeutronAPIRouteTimeout string `json:"neutronAPIRouteTimeout"`
```

Comment:
I wonder if need the field at all. I think with the default values expanded to include the necessary annotations, openstack-operator already sets 120s for the route object via override field. And a user then can go and edit the value for a different one, if needed (I tried with 130s and it worked). On the other hand, this field seems to be ignored by openstack-operator.

Why is setting a custom value through override section for the route in CP CR not enough?

Reply:
Done!